scripts/version/Version.props

```diff
@@ -7,6 +7,6 @@
     <FullVersion>$(MainVersion).$(BuildNumber)</FullVersion>
     <AssemblyVersion>$(MainVersion)</AssemblyVersion>
     <AssemblyFileVersion>$(FullVersion)</AssemblyFileVersion>
-    <AssemblyInformationalVersion>Version:$(FullVersion) Branch:$(BranchName) Sha1:$(Sha1)</AssemblyInformationalVersion>

```

Comment:
This is a hack.
This file is used by `ChangeVersion.proj`, which re-populates the `AssemblyInfo.Shared.cs` in the pipeline.
I plan to revert this with my next PR, but for now all it does it set the version of this property to `9.0.0-rc ...`.